OpenFilesScreen.c

```diff
@@ -201,6 +202,22 @@ static OpenFiles_ProcessData* OpenFilesScreen_getProcessData(pid_t pid) {
       pdata->error = WEXITSTATUS(wstatus);
    }
 
+   /* On linux, `lsof -o -F` omits SIZE, so add it back. */
+   /* On macOS, `lsof -o -F` includes SIZE, but overwrite it for consistency. */
+   fdata = pdata->files;
+   size_t sizeIndex = getIndexForType('s');
+   size_t filenameIndex = getIndexForType('n');
+   while (fdata) {
+      item = &fdata->data;
+      struct stat st;
+      if (stat(item->data[filenameIndex], &st) == 0) {
+         char sizeChar[21]; /* 20 (long long) + 1 (NULL) */
```

Comment:
`sizeChar` is not a good identifier for this variable. Something a bit more descriptive like `fileSizeBuf` or `fileSizeStr` is IMHO better.

Reply:
I changed this.  Thanks for all the reviews to try to make this code decent!  I should have noticed that this variable name didn't fit in earlier.